core/src/main/java/hudson/Main.java

```diff
@@ -83,7 +86,8 @@ private static String getHudsonHome() {
     }
 
     /**
-     * Run command and place the result to a remote Hudson installation
+     * Run command and send result to {@code ExternalJob} in the {@code external-monitor-job} plugin.
+     * Obsoleted by {@code SetExternalBuildResultCommand} but kept here for compatibility.
      */
     public static int remotePost(String[] args) throws Exception {
```

Comment:
Right, they both call some other APIs, called also by the REST endpoint this invokes.

I did not particularly care to deprecate it since it is unlikely anyone was trying to call it as a Java API to begin with. I was more concerned with making it apparent what this code was for to someone reading Jenkins core sources, who might be very confusing by a `main` method which does not in fact start Jenkins!